skins/laika/src/components/pages/Contact.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+	<div class="column is-full contact-form">
+		<h1 class="title">{{ $t( 'contact_form_title' ) }}</h1>
+		<span class="help is-danger has-padding-bottom-18" v-if="contactData.errors">{{ $t('contact_form_error') }}</span>
+		<form method="post" action="/contact/get-in-touch" @submit="submit" id="laika-contact">
+			<fieldset>
+				<div>
```

Comment:
unnecessary `div`

Reply:
All the other input elements and their labels / errors have this `div` so it's there for consistency's sake I would argue.

Reply:
It literally  doesn't do anything :D